be/extmap.c

```diff
@@ -956,6 +1114,10 @@ static void emap_it_init(struct m0_be_emap_cursor *it,
 
 static void be_emap_close(struct m0_be_emap_cursor *it)
 {
+	if(it->ec_recbuf.b_addr != NULL ) {
+	   m0_buf_free(&it->ec_recbuf);
```

Comment:
Shall we do , it->ec_recbuf.b_addr = NULL after freeing ?

Reply:
m0_buf_free already sets the pointer to NULL